tests/validation/tests/v3_api/common.py

```diff
@@ -878,6 +878,14 @@ def check_if_ok(url, verify=False, headers={}):
         print("Connection Error - " + url)
         return False
 
+def retry_cmd(pod, cmd, expected,retry_count=5):
```

Comment:
Why is this a count of 5? Can we not use any existing timeouts defined in this python file here?